app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] From the specifications, the comment belongs to an `author`, not `user`. Now, since we don't have an Author model we need to specify the model being referred to here, which is the User model.

  ```
  belongs_to :author, class_name: 'User'
  ```

Reply:
done